timetracker/entries/urls.py

```diff
@@ -4,9 +4,10 @@
 
 
 urlpatterns = [
-    url(r'^clients/$', views.clients, name='client-list'),
+	url(r'^$', views.RootRedirectView.as_view(), name='root'),
```

Comment:
You should be careful with using tabs and spaces for indentation. Recommendation is to use 4 spaces.